src/Gateway/Blocks/PayplugOney.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace Payplug\PayplugWoocommerce\Gateway\Blocks;
+
+class PayplugOney extends PayplugGenericBlock {
+
+	/**
+	 * Payment method name/id/slug.
+	 *
+	 * @var string
+	 */
+	protected $name = "oney";
+
+	protected $icon = '';
+
+	protected $cart;
+
+	protected $total_price;
+
+
+	public function oney_enabled() {
```

Comment:
this should be name `get_payment_method_data` shouldn't it?

Reply:
PayplugOney is oney a parent class, the name property is being deprecated by the children classes